src/main/resources/reference.conf

```diff
@@ -117,4 +120,10 @@ videobridge {
     # such candidates will be ignored.
     resolve-remote-candidates = false
   }
+
+  dtls {
+    # packets from the egress end-up in this queue, right before encryption. Its
```

Comment:
This isn't right - the DtlsTransport doesn't actually do the encryption of the SRTP/SRTCP.

Reply:
right, so this is right after encryption then if i'm reading the code correctly the pipeline is as simple as writing to the socket?

Reply:
Yes, I believe that's right.  (It nonetheless takes up 15% of bridge CPU time...)

Reply:
I've updated the description, please let me know if you feel it needs any other changes.